app/frontend/styles/components/upload-documents.scss

```diff
@@ -0,0 +1,19 @@
+.create-title h1 {
+  margin-bottom: 0;
+}
+
+.create-title h5 {
+  font-weight: 500;
+}
+
+.create-title .govuk-back-link {
+  margin-bottom: 0;
+}
+
+.upload-group .govuk-button {
+  margin-bottom: 0;
+}
+
+.hidden_input {
+  display: none;
```

Comment:
Class names inconsistent (e.g. hidden-input)

Reply:
Thanks for spotting!

Reply:
Fixed in 6747a148